doc/source/whatsnew/v1.1.0.rst

```diff
@@ -13,6 +13,20 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
+.. _whatsnew_110.specify_missing_labels:
+
+KeyErrors raised by loc specify missing labels
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Previously, if labels were missing for a loc call, a Key Error was raised stating that this was no longer supported.
+
+Now the error message also includes a list of the missing labels. For example,
+
+.. code-block:: ipython
+
+    s = pd.Series({"a": 1, "b": 2, "c": 3})
+    s.loc[["a", "b", "missing_0", "c", "missing_1", "missing_2"]]
```

Comment:
you don’t need a section for this just a note in the whats new

Reply:
sure, I've removed the code block and just left the text entry in the what's new.